tests/runner.py

```diff
@@ -100,6 +100,8 @@ def delete_contents(pathname):
 
 EMTEST_SKIP_SLOW = os.getenv('EMTEST_SKIP_SLOW')
 
+EMTEST_NO_NATIVE_CLANG = os.getenv('EMTEST_NO_NATIVE_CLANG')
```

Comment:
we have `EMTEST_LACKS_SOUND_HARDWARE` and a few others in browser, so perhaps this could be `EMTEST_LACKS_NATIVE_CLANG`